content/en/post/murakami-ia2023/index.md

```diff
@@ -0,0 +1,33 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "Mr. Murakami presented his research at Internet Architecture workshop."
+subtitle: ""
+summary: ""
+authors: ["kyoya-murakami"]
+tags: ["Kubernetes"]
+categories: []
+date: 2023-03-17T16:05:10+09:00
+lastmod: 2023-03-17T16:05:10+09:00
+featured: false
+draft: false
+
+# Featured image
+# To use, add an image named `featured.jpg/png` to your page's folder.
+# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
+image:
+  caption: ""
+  focal_point: ""
+  preview_only: false
+
+# Projects (optional).
+#   Associate this post with one or more of your projects.
+#   Simply enter your project's folder or file name without extension.
+#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
+#   Otherwise, set `projects = []`.
+projects: []
+---
+Kyoya Murakami, M2 in our laboratory, gave his research at [Internet Architecture workshop](https://ken.ieice.org/ken/program/index.php?tgs_regid=4c4585ce7890559d709b0336010193f3fb3b287a5d87e17088da1225d91b4e66&tgid=IEICE-IA).
+
+
+He introduced his research titled "Evaluation of a load balancing method considering inter-site latency" that evaluation of a load balancing method for Kubernetes in edge computing.
```

Comment:
He introduced his research titled "Evaluation of a load balancing method considering inter-site latency", which evaluated a load balancing method for Kubernetes in edge computing environments, and received a Student Research Award.